featurebyte/worker/task/batch_feature_create.py

```diff
@@ -0,0 +1,152 @@
+"""
+Batch feature create task
+"""
+from __future__ import annotations
+
+from typing import Any, Iterator, cast
+
+import asyncio
+import concurrent.futures
+import os
+from contextlib import contextmanager
+
+from bson import ObjectId
+
+from featurebyte.exception import DocumentInconsistencyError
+from featurebyte.logging import get_logger
+from featurebyte.models.base import activate_catalog
+from featurebyte.models.feature import FeatureModel
+from featurebyte.schema.worker.task.batch_feature_create import BatchFeatureCreateTaskPayload
+from featurebyte.service.feature import FeatureService
+from featurebyte.worker.task.base import BaseTask
+
+logger = get_logger(__name__)
+
+
+@contextmanager
+def set_environment_variable(variable: str, value: Any) -> Iterator[None]:
+    """
+    Set the environment variable within the context
+
+    Parameters
+    ----------
+    variable: str
+        The environment variable
+    value: Any
+        The value to set
+
+    Yields
+    ------
+    Iterator[None]
+        The context manager
+    """
+    previous_value = os.environ.get(variable)
+    os.environ[variable] = value
+
+    try:
+        yield
+    finally:
+        if previous_value is not None:
+            os.environ[variable] = previous_value
+        else:
+            del os.environ[variable]
+
+
+async def execute_sdk_code(catalog_id: ObjectId, code: str) -> None:
+    """
+    Activate the catalog and execute the code in server mode
+
+    Parameters
+    ----------
+    catalog_id: ObjectId
+        The catalog id
+    code: str
+        The SDK code to execute
+    """
+    # activate the correct catalog before executing the code
+    activate_catalog(catalog_id=catalog_id)
+
+    # execute the code
+    with set_environment_variable("SDK_EXECUTION_MODE", "SERVER"):
```

Comment:
This is not required for the current PR. It will be required once we have different behavior between running feature save in client side versus server side.